src/main.rs

```diff
@@ -847,8 +890,18 @@ fn bisect_nightlies(cfg: &Config, client: &Client) -> Result<BisectionResult, Er
     let mut last_failure = if let Some(Bound::Date(date)) = cfg.args.end {
         date
     } else {
-        today
+        if let Some(date) = Toolchain::default_nightly() {
+            // start date must be prior to end date
```

Comment:
Could we print something like "Using {date} from the currently installed default nightly toolchain as the last failure" or something like that?